src/unity/thunks/index.ts

```diff
@@ -0,0 +1,127 @@
+// Libraries
+import {Dispatch} from 'react'
+
+// API
+import {
+  getOrgsInvites,
+  getOrgsUsers,
+  resendOrgInvite,
+  deleteOrgInvite,
+  deleteOrgUser,
+} from 'src/unity/api'
+
+// Actions
+import {
+  setAll,
+  updateUser,
+  updateInvite,
+  removeInvite,
+  removeUser as removeUserAction,
+  removeUserStatus,
+  removeInviteStatus,
+  resendInviteStatus,
+  Action,
+} from '../reducers'
+import {RemoteDataState} from '@influxdata/clockface'
+
+// Types
+import {Invite, CloudUser as User} from 'src/types'
+
+// Constants
+import {GTM_USER_REMOVED} from 'src/unity/constants'
+
+export const getUsersAndInvites = async (dispatch: Dispatch<Action>) => {
+  try {
+    const userResp = await getOrgsUsers()
+    const inviteResp = await getOrgsInvites()
```

Comment:
Should this be `const [userResp, inviteResp] = await Promise.all([getOrgsUsers(), getOrgsInvites()])` to parallelize the two requests?

Reply:
Great catch!  I'll change that.